nautobot/extras/tests/integration/test_tagfilter.py

```diff
@@ -30,9 +31,10 @@ def tearDown(self):
 
     def test_tag_matching_content_type(self):
         # Navigate to the Provider list view
-        self.browser.links.find_by_partial_text("Circuits").click()
-        self.browser.links.find_by_partial_text("Providers").click()
-
+        self.browser.links.find_by_text("Circuits").click()
+        time.sleep(0.5)
+        self.browser.links.find_by_text("Providers").click()
+        time.sleep(0.5)
```

Comment:
Changed to `find_by_text` and added sleep because self.browser was still on the Home Page.

Reply:
But the test still fails in the CI, despite it passes in the local dev env.

```
inv tests -k -s -l nautobot.extras.tests.integration.test_tagfilter.TagFilterTestCase.test_tag_matching_content_type --tag integration

<...>
System check identified 1 issue (0 silenced).
.
----------------------------------------------------------------------
Ran 1 test in 31.945s

OK
Emptying test database "default"...
Database default emptied!
Preserving test database for alias 'default'...
```